src/415D90.c

```diff
@@ -257,16 +259,7 @@ BSS s32 D_802ACC64;
 BSS s32 D_802ACC68;
 BSS s32 D_802ACC6C;
 BSS s32 D_802ACC70[24];
-BSS s32 D_802ACCD0;
-BSS s32 D_802ACCD4[0xC5]; // unused?
-BSS s32 D_802ACFE8;
-BSS s32 D_802ACFEC;
-BSS s32 D_802ACFF0;
-BSS s32 D_802ACFF4;
-BSS s32 D_802ACFF8;
-BSS s8 D_802ACFFC;
-BSS s8 D_802ACFFD;
-BSS s16 D_802ACFFE; // unused?
+BSS PopupMenu D_802ACCD0;
```

Comment:
very nice to see

Reply:
yeah but the function that uses it is still a mystery :/ it loads some of the fields directly in a way that doesn't make sense